applications/vanilla/controllers/class.postcontroller.php

```diff
@@ -172,17 +172,22 @@ public function discussion($CategoryUrlCode = '') {
 
         touchValue('Type', $this->Data, 'Discussion');
 
-        // See if we should hide the category dropdown.
-        if ($this->ShowCategorySelector) {
+        if (!$UseCategories || $this->ShowCategorySelector) {
```

Comment:
The config is still used. We need to either no use it anywhere or respect it until we remove it for real.